<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>RECOMB 2017 - Accepted Posters</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/stylish-portfolio.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
</head>
<body>
	<!-- Navigation -->
	<a id="schedule-shortcut" href="schedule.html" class="btn btn-dark btn-lg toggle"><i class="fa fa-calendar"></i></a>
    <a id="menu-toggle" href="#" class="btn btn-dark btn-lg toggle"><i class="fa fa-bars"></i></a>
    <nav id="sidebar-wrapper">
        <ul class="sidebar-nav">
            <a id="menu-close" href="#" class="btn btn-light btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
            <li>
                <a href="index.html" onclick=$("#menu-close").click();>RECOMB Home</a>
            </li>
            <li>
                <a href="index.html#about" onclick=$("#menu-close").click();>About</a>
            </li>
            <li>
                <a href="index.html#services" onclick=$("#menu-close").click();>Overview</a>
            </li>
            <li class="dropdown">
                <a href="registration.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Register <span class="caret"></span></a>
                <ul class="dropdown-menu" role="menu">
                        <li><a href="travelfellowship.html">Travel Fellowship</a></li>
                        <li><a href="accommodation.html">Hotel Information</a></li>
                </ul>
            </li>
                        <li>
                <a href="index.html#portfolio" onclick=$("#menu-close").click();>Submit to RECOMB</a>
            </li>
            <li class="dropdown">
                <a href="program.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Program <span class="caret"></span></a>
                <ul class="dropdown-menu" role="menu">
                        <li><a href="schedule.html">Schedule</a></li>
                        <li><a href="accepted_papers.html">Accepted Papers</a></li>
                        <li><a href="#">Accepted Posters</a></li>
			<li><a href="awards.html">Award Winners</a></li>
                        <li><a href="useful_links.html">Useful Links</a></li>
                </ul>
            </li>
			<li>
                <a href="keynote.html" onclick=$("#menu-close").click();>Keynotes</a>
            </li>
			<li>
                <a href="committees.html" onclick=$("#menu-close").click();>Committees</a>
            </li>
		<li class="dropdown">
                <a href="satellites.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Satellites <span class="caret"></span></a>
                        <ul class="dropdown-menu" role="menu">
                        <li><a href="recomb-seq.html">RECOMB-Seq</a></li>
                        <li><a href="http://computationalgenomics.bioinformatics.ucla.edu/2017-recomb-satellite-meetings/" target=_blank>RECOMB-Genetics</a></li>
                        <li><a href="http://computationalgenomics.bioinformatics.ucla.edu/2017-recomb-satellite-meetings/" target=_blank>RECOMB-CCB</a></li>
                </ul>
            </li>
            <li>
                <a href="index.html#contact" onclick=$("#menu-close").click();>Venue</a>
            </li>
		<li><br /><center><img src="img/logo-banner.png" alt="" width="95%"></center></li>
        </ul>
    </nav>
	<header class="main-header">
		<div class="text-vertical-center">
            <font color=white>
			<h1>RECOMB 2017</h1>
            <h2>The 21st Annual International Conference on Research in Computational Molecular Biology</h2>
			<h3>Hong Kong, May 3-7, 2017</h3>
			<h4>(May 3: Registration &amp; Reception, May 4-7: Talks)</h4>
			</font>
        </div>
	</header>
	<!--Body-->
	<div class="container">
		<h3>Accepted Posters</h3>
		
		<p align="justify"><font color=red><i>Guidelines for poster presentation can be found <a href="data/poster-guidelines.pdf">here</a></i>. <i>Poster abstracts can be downloaded <a href="data/RECOMB2017_poster_book-3.pdf">here</a></i>.</font></p>
	<ol>		
<li>Jiaan Dai, Fengchao Yu, Ning Li and Weichuan Yu<br /><b> Is it feasible to only use tags in database search for PTM-invariant peptide identification? --- A simulation-based study</b></li>
<li>Fengchao Yu, Ning Li and Weichuan Yu<br /><b> ECL 2.0: Exhaustively Identifying Cross-Linked Peptides with a Linear Computational Complexity</b></li>
<li>Jo&atilde;o Herminio Martins Da Silva, Disraeli Vasconcelos, Lia Pinho and Beatriz Chaves<br /><b> De novo Receptor-Based Design of new Integrin Beta 1 Inhibitors and in silico ADMET methods evaluation</b></li>
<li>Natalie Fox, Emilie Lalonde, Julie Livingstone, Julia Hopkins, Yu-Jia Shiah, Vincent Huang, Takafumi Yamaguchi, Veronica Sabelnykova, Lawrence Heisler, Michael Fraser, Theodorus van der Kwast, Robert Bristow and Paul Boutros<br /><b> Integrated somatic subtypes of localized prostate cancer with prognostic implications</b></li>
<li>Robert Brown, Eleazar Eskin and Bogdan Pasaniuc<br /><b> Haplotype-based eQTL Mapping Increases Power to Identify eGenes</b></li>
<li>Thahmina Ali, Baekdoo Kim, Carlos Lijeron and Konstantinos Krampis<br /><b> Xplore-seq: a comprehensive, standardized and scalable method to transcriptomic profiling as a clinically-useful application</b></li>
<li>Serghei Mangul, Igor Mandric, Alex Zelikovsky and Eleazar Eskin<br /><b> Profiling adaptive immune repertoires across multiple human tissues by RNA Sequencing</b></li>
<li>Jing Qin, Yaohua Hu, Jen-Chih Yao, Yiming Qin, Ka Hou Chu and Junwen Wang<br /><b> Group Sparse Optimization: An Integrative OMICs Method to Predict Master Transcription Factors for Cell Fate Conversion</b></li>
<li>Jennifer Zou, Farhad Hormozdiari, Jason Ernst, Jae-Hoon Sul and Eleazar Eskin<br /><b> Leveraging allele-specific expression to improve fine-mapping for eQTL studies</b></li>
<li>Luca Alessandrì and Raffaele Calogero<br /><b> CASC: Classification Analysis of Single Cell Sequencing Data</b></li>
<li>Nan Papili Gao, S.M Minhaz Ud-Dean and Rudiyanto Gunawan<br /><b> SINCERITIES: Inferring gene regulatory net-works from time-stamped single cell transcrip-tional expression profiles</b></li>
<li>Feng Yang, Wei Zeng, Ke Liu, Guangbin Wang, Zhengwen Li, Keli Huang and Nini Rao<br /><b> The Roles of Signal Pathways mediated by AF-VHD-related microRNA combinations in Development from VHD to AF-VHD</b></li>
<li>Egor Dolzhenko, Joke J.F.A. van Vugt, Subramanian S. Ajay, Ryan Taft, David R. Bentley, Jan H. Veldink and Michael A. Eberle<br /><b> Detection of long repeat expansions from PCR-free whole-genome sequence data</b></li>
<li>Andreas Dwi Maryanto Gunawan, Bingxin Lu, Hon Wai Leong and Louxin Zhang<br /><b> Computer Program for Verification of Phylogenetic Networks</b></li>
<li>Vera Kaiser and Colin Semple<br /><b> Diversity of mutational loads at CTCF binding sites predicts foci of dysregulated expression</b></li>
<li>Dong-Jun Lee and Hyeon-So Gi<br /><b> Comparative transcriptome analysis of resistant and susceptible Korea rice genotype in response to bakanae disease</b></li>
<li>Thrasyvoulos Karydis, Aditya Khosla, Manolis Kellis and Joseph M. Jacobson<br /><b> Learning hierarchical motif representations for protein analysis, search and design</b></li>
<li>Jie Ren, Fengzhu Sun, Nathan A Ahlgren, Jed A Fuhrman and Yang Young Lu<br /><b> VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data</b></li>
<li>Dat Duong, Lisa Gai, Sagi Snir, Eun Yong Kang, Buhm Han, Jae Hoon Sul and Eleazar Eskin<br /><b> Applying meta-analysis to Genotype-Tissue Expression data from multiple tissues to identify eQTLs and increase the number of eGenes</b></li>
<li>Ruban Durairaj D, Murugesh Easwaran and Shanmughavel P<br /><b> Molecular docking and dynamics simulation studies of Antitubercular compounds with Enoyl-ACP reductase enzyme of MDR-Tuberculosis</b></li>
<li>Luming Meng, Wenjun Xie, Sirui Liu, Ling Zhang and Yiqin Gao<br /><b> Chromatin modeling reveals segregation of chromosome characteristics and fuctions in 3D space</b></li>
<li>Harry Taegyun Yang, Serghei Mangul, Noah Zaitlen, Sagiv Shifman and Eleazar Eskin<br /><b> Repeat Elements Profile Across Different Tissues in GTEx Samples</b></li>
<li>Ung-Han Yoon, Jaecheol Jeong, Jang-Ho Hahn, Jung-Wook Yang, Tae-Ho Kim, Hyeong-Un Lee, Young-Ju Seol, Sang-Sik Nam, Sang-Soo Kwak and Tae-Ho Lee<br /><b> Structural and evolutionary analysis of sweetpotato chloroplast genomes</b></li>
<li>Roven Rommel Fuentes, Dmytro Chebotarov, Sean Smith, Jorge Duitama, Marghoob Mohiyuddin, Ramil Mauleon, Andrey Grigoriev, Rod Wing and Nickolai Alexandrov<br /><b> Discovery of Structural Variants in 3,024 Rice Genomes</b></li>
<li>Magdalena Strauss, Lorenz Wernisch and John Reid<br /><b> New MCMC methods for pseudo-time estimation using Gaussian processes</b></li>
<li>Yali Xiao, Ssu-Min Fang, Yun-Hsin Tsou, Cheng-Fang Tsai and Pei-Chun Chang<br /><b> The Potential Inhibitors in Chinese Traditional Medicine for Bcr-AblT315I Mutation of Chronic Myelogenous Leukemia</b></li>
<li>Chalida Rangsiwutisak, Treenut Saithong and Saowalak Kalapanulak<br /><b> Exploring the landscape of regulatory elements on non-coding regions of cassava genome via motif-based screening</b></li>
<li>Zhun Miao and Xuegong Zhang<br /><b> A new differential expression analysis method for single-cell RNA-Seq data</b></li>
<li>Robert Schöpflin and Martin Vingron<br /><b> Interactive visualization of Hi-C data and epigenetic marks</b></li>
<li>Sang Chun, Caitlin Rodriguez, Ryan Mills and Peter Todd<br /><b> Regulation of Differentiation in SH-SY5Y Neuroblastoma Cells by Translation of Upstream Open Reading Frames</b></li>
<li>Jian-Ying Chiu, Tse-Ching Ho, Syun-Wun Liang and Yen-Hua Huang<br /><b> Quality assessment of genome assembly generated by the third-generation sequencing platform PacBio and an online editor for continuous update of genome annotation</b></li>
<li>Michael Altenbuchinger, Philipp Schwarzfischer, Thorsten Rehberg, J&ouml;rg Reinders, Christian W. Kohler, Wolfram Gronwald, Julia Richter, Monika Szczepanowski, Neus Masqué-Soler, Wolfram Klapper, Pete J. Oefner and Rainer Spang <br/><b> Molecular signatures that can be transferred
across different omics platforms</b></li>
<li>Xin Li, Chong Chu, Jingwen Pei, Ion Mandoiu and Yufeng Wu<br /><b> CircMarker: A Fast and Accurate Algorithm for Circular RNA Detection</b></li>
<li>Lisa Gai, Dat Duong and Eleazar Eskin<br /><b> Finding associated variants in genome-wide associations studies on multiple traits</b></li>
<li>Gongchao Jing, Jian Xu and Xiaoquan Su<br /><b> Microbiome Search Engine: Enabling Rapid Microbiome Samples Search In Large-scale Database</b></li>
<li>Diya Sen, Mathu Malar Chandrababunaidu and Sucheta Tripathy<br /><b> An efficient contamination removal tool for improving prokaryotic genome assembly</b></li>
<li>Dan Huang, Fu Yan Hu and Nelson Tang<br /><b> Splicing aberration of TP53 transcripts in cancers: mechanisms and effects</b></li>
<li>Alexey Morgunov, Alexander Gunnarsson, Norbert Fehér and Madan Babu<br /><b> Investigating residue coevolution in proteins from a structural perspective</b></li>
<li>Gregory Smith, Alan Stern and Marc Birtwistle<br /><b> Inferring Dynamic Network Architecture from Time Series Perturbation Data</b></li>
<li>Daniel Hogan, Bharathi Vellalore, Ronald Geyer and Anthony Kusalik<br /><b> SeqMiner: A Weka package for mining phage display sequence data</b></li>
<li>Chao Sima, Shanaz Ghandhi, Sally A. Amundson, David J. Brenner and Michael Bittner<br /><b> Feature reduction for practical radiation biodosimetry using combined approach of gene co-expression network and pathway knowledge</b></li>
<li>Kimberly Mackay, Christopher Eskiw and Anthony Kusalik<br /><b> 3D Yeast Genome Prediction with Constraint Logic Programming</b></li>
<li>Jae Hyeon Oh, Tae Ho Lee, Chang Kug Kim, Lee Dong-Jun and Dowan Kim<br /><b> Transcripsome and metabolism studies to identify alkaloid biosynthesis genes in poppies</b></li>
<li>Wendao Liu, Zhun Miao and Xuegong Zhang<br /><b> Assessing mRNA integrity of single-cell RNA-Seq data using mRIN</b></li>
<li>Alden King-Yung Leung, Nana Jin, Kevin Y. Yip and Ting-Fung Chan<br /><b> OMView: a comprehensive visualization suite for interpreting optical mapping data</b></li>
<li>Heng Xiong, Dongbing Liu and Leo Lee<br /><b> RED-ML: a novel, effective RNA editing detection method based on machine learning</b></li>
<li>Dawn Chen and Ming-Jing Hwang<br /><b> Reducing Runtime in CONTRAlign by Feature Reduction</b></li>
<li>Yu-Chieh Liao and Hsin-Hung Lin<br /><b> drVM: detect and reconstruct known viral genomes from metagenomes</b></li>
<li>Kuei-Ling Sun, Onkar Singh and Emily Chia-Yu Su<br /><b> Predicting Allergens and Identifying Interpretable Allergenic Biological Features using Machine Learning Algorithms</b></li>
<li>Xiang-Tian Yu, Jin-Song Zhang, Shao-Yan Sun, Tao Zeng and Luonan Chen<br /><b> Individual-specific edge network analysis for disease prediction</b></li>
<li>Juan Zhao, Yiwei Zhou, Xiujun Zhang and Luonan Chen<br /><b> Part mutual information for quantifying direct associations in networks</b></li>
<li>Dowan Kim, Jae-Hyeon Oh, Dong-Jun Lee, Chang-Kug Kim and Tae-Ho Lee<br /><b> Transcriptome analysis in some varieties of Papaver rhoeas</b></li>
<li>Hojin Gwak and Mina Rho<br /><b> ABCProfiler: a program for Alignment Based Clustering and taxonomy Profiling</b></li>
<li>Jehyun Jeon and Mina Rho<br /><b> Modular typeⅠPKS cluster in sediment microbiomes</b></li>
<li>Julian Zaugg, Yosephine Gumulya and Mikael Bodén<br /><b> Learning From Sequence-Activity Data - Predicting Enantioselectivity of an Epoxide Hydrolase</b></li>
<li>Julian Zaugg, Yosephine Gumulya, Mikael Bod&eacute;n and Alpeshkumar Malde<br /><b> Using Theory to Reconcile Experiments: Understanding the Origin of Enantioselectivity of Epoxide Hydrolase</b></li>
<li>Ingoo Lee and Hojung Nam<br /><b> Identification of drug-target interactions using weighted interactome network</b></li>
<li>Kavya Vaddadi, Naveen Sivadasan, Kshitij Tayal and Rajgopal Srinivasan<br /><b> Sequence Alignment on Directed Graphs</b></li>
<li>Naveen Sivadasan, Rajgopal Srinivasan and Kshama Goyal<br /><b> Kmerlight: fast and accurate k-mer abundance estimation</b></li>
<li>Kang Ning, Shaojun Yu and Maozhen Han<br /><b> Species-species network analysis for microbial communities</b></li>
<li>Przemyslaw Szalaj, Zhonghui Tang, Yijun Ruan and Dariusz Plewczynski<br /><b> Three-dimensional genome modeling based on ChIA-PET data</b></li>
<li>Mika Gustafsson<br /><b> LASSIM - a network inference toolbox for genome-wide mechanistic modeling</b></li>
<li>Saad Mneimneh and Syed Ali Ahmed<br /><b> Combinatorial and Probabilistic Aspects of the Multiple RNA Interaction Problem</b></li>
<li>Bingxin Lu and Hon Wai Leong<br /><b> GI-Cluster: detecting genomic islands in newly sequenced microbial genomes via consensus clustering on multiple features</b></li>
<li>Wanatsanan Siriwat, Saowalak Kalapanulak and Treenut Saithong<br /><b> Investigation of canonical metabolic network of plants through topology-based analysis</b></li>
<li>Shah Md. Shahik, Md. Saiful Islam, Md Sohel and Mohd.Omar Faruk Sikder<br /><b> Mining the Proteome of Fusobacterium nucleatum subsp. nucleatum ATCC 25586 for Potential Therapeutics Discovery: An In Silico Approach</b></li>
<li>Abhishek Das, Subhadeep Das, Samrat Ghosh and Sucheta Tripathy<br /><b> Classification of promoter and enhancer pairs based on expression patterns for building a predictive model</b></li>
<li>Alexander Shlemov, Alexey Gurevich, Alla Mikheenko, Anastasiia Abramova, Anton Korobeynikov, Hosein Mohimani and Pavel Pevzner<br /><b> Identification of novel peptidic antibiotics by searching large-scale mass spectra against natural products databases</b></li>
<li>Heeju Noh, Ziyi Hua and Rudiyanto Gunawan<br /><b> Identifying molecular targets of drugs from gene transcriptional profiles</b></li>
<li>Kui Hua and Xuegong Zhang<br /><b> Modeling genome coverage and estimating genome length in metagenomics</b></li>
<li>Jongsoo Keum and Hojung Nam<br /><b> Constructing prediction of drug-target interaction model using deep neural network approach</b></li>
<li>Sajad Mirzaei and Yufeng Wu<br /><b> RENT+: An Improved Method for Inferring Local Genealogical Trees from Haplotypes with Recombination</b></li>
<li>Mohammadhossein Moeinzadeh, Jun Yang and Martin Vingron<br /><b> De novo assembly and haplotyping of Sweet Potato (Ipomoea Batatas [L</b></li>
<li>Jingwen Pei and Yufeng Wu<br /><b> STELLS2: Fast and Accurate Coalescent-based Maximum Likelihood Inference of Species Trees from Gene Tree Topologies</b></li>
<li>Xintong Chen, David Chiang, M. Cecilia Berlin and Bojan Losic<br /><b> Single cell RNA-seq immunodynamics in peanut allergy</b></li>
<li>Chirag Jain, Luis M. Rodriguez, Alexander Dilthey, Adam Phillippy, Kostas Konstantinidis and Srinivas Aluru<br /><b> Breaking the Scalability Barrier for Core-genome Identity Computation</b></li>
<li>Chong Chu, Jingwen Pei and Yufeng Wu<br /><b> An improved approach for reconstructing consensus repeats from short sequence reads</b></li>
<li>Chong Chu, Xin Li and Yufeng Wu<br /><b> GAPPadder: A Sensitive Approach for Closing Gaps on Draft Genomes with Short Sequence Reads</b></li>
<li>Chen Yang, Readman Chiu, Daniel MacMillan, Zhuyi Xue, Rene Warren and Inanc Birol<br /><b> Differential Expression Analysis of Alternatively Polyadenylated 3’UTR using RNA-Seq</b></li>
<li>Cenk Sahinalp<br /><b> The Cancer Genome Collaboratory</b></li>
<li>Alexander Shlemov, Sergey Bankevich, Andrey Bzikadze and Yana Safonova<br /><b> Antibody repertoire construction for organisms with unknown germline V/J genes</b></li>
<li>Sergey Bankevich, Alexander Shlemov, Andrey Bzikadze and Yana Safonova<br /><b> Antibody repertoire construction from short HiSeq Rep-Seq reads</b></li>
<li>Sergey Bankevich, Alexander Shlemov, Andrey Bzikadze and Yana Safonova<br /><b> Antibody repertoire construction from Ion Torrent Rep-seq reads</b></li>
<li>Andrey Bzikadze, Sergey Bankevich, Alexander Shlemov and Yana Safonova<br /><b> AntEvolo: a novel approach for clonal analysis of antibody repertoires</b></li>
<li>Yue Wu, Eleazar Eskin and Sriram Sankararaman<br /><b> Improving imputation by maximizing power</b></li>
<li>Rayan Chikhi, Charles Deltel, Guillaume Rizk, Claire Lemaitre, Pierre Peterlongo, Kristoffer Sahlin, Lars Arvestad, Paul Medvedev and Dominique Lavenier<br /><b> High-quality, fast, and memory-efficient assembly of metagenomes and large genomes using Minia-pipeline</b></li>
<li>Luiz Carlos Irber Junior, C. Titus Brown and Tim Head<br /><b> Decentralized indexes for public genomic data</b></li>
<li>Andrzej Kloczkowski, Girik Malik and Anirban Banerji<br /><b> Deciphering general characteristics of residues constituting allosteric communication paths</b></li>
<li>Sheida Hayati and Antonina Mitrofanova<br /><b> Computational approaches to identify alternative splice variants as biomarkers of disease progression and drug resistance in Acute Myeloid Leukemia</b></li>
<li>Praveen Kumar, Murugesh Easwaran, Nilavamuthan Chandrasekaran and Shanmughavel Piramanayagam<br /><b> Molecular Insights of Peptide Folding Propensities for Cancer Drug Target Improvisation and Anti Microbial Peptide Library</b></li>
<li>R. Gonzalo Parra, Nikolaos Papadopoulos, Laura Ahumada-Arranz and Johannes Soeding<br /><b> scTree: reconstructing complex cellular lineage trees from single-cell RNA-seq data</b></li>
<li>Eric Ho<br /><b> IDICAP2: An Improved Tool for Integrating Drug Intervention Based on Cancer Panel</b></li>
<li>Pravin Dudhagara, Anjana Ghelani and Rajesh Patel<br /><b> Comparative extreme microbiomes exploration using bioinformatics methodologies</b></li>
<li>Weronika Wronowska, Krzysztof Gogolewski, Bogdan Lesyng and Anna Gambin<br /><b> PARP inhibition cytotoxicity or cytoprotection - inferring molecular pathways heterogeneity from transcriptional data</b></li>
<li>Nazar Zaki and Chandana Tennakoon<br /><b> BioCarian: A Search Engine for Performing Exploratory Searches of Biological Databases</b></li>
<li>Sunhye Park, Young Chan Park, Kiejung Park and In-Song Koh<br /><b> A comprehensive variation analysis based on whole-genome of 62 Koreans and constructing Korean variome database called as HYKVB</b></li>
<li>Camille Scott and C. Titus Brown<br /><b> Characterizing RNA-seq assembly graphs: when is enough, enough?</b></li>
<li><i>Retracted</i></li>
<li>Jong Wha J. Joo, Farhad Hormozdiari, Jeahoon Sul, and Eleazar Eskin<br /><b> Model-based multiple variants test considering causal status</b></li>
	</ol>
	</div>
	<footer>
        <div class="container">
            <div class="row">
                <div class="col-lg-10 col-lg-offset-1 text-center">
                    <hr class="small">
                    <p class="text-muted">Copyright &copy; RECOMB 2017 | Theme: Bootstrap Stylish Portfolio | Design: Sumaiya Nazeen</li>
                </div>
            </div>
        </div>
        <a id="to-top" href="#top" class="btn btn-dark btn-lg"><i class="fa fa-chevron-up fa-fw fa-1x"></i></a>
    </footer>
	<!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Custom Theme JavaScript -->
    <script>
	$('ul.sidebar-nav li.dropdown').hover(function() {
	  $(this).find('.dropdown-menu').stop(true, true).delay(200).fadeIn(500);
	}, function() {
	  $(this).find('.dropdown-menu').stop(true, true).delay(200).fadeOut(500);
	});
    </script>	
    <script>
    // Closes the sidebar menu
    $("#menu-close").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });
    // Opens the sidebar menu
    $("#menu-toggle").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });
    // Scrolls to the selected menu item on the page
    $(function() {
        $('a[href*=#]:not([href=#],[data-toggle],[data-target],[data-slide])').click(function() {
            if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
                var target = $(this.hash);
                target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
                if (target.length) {
                    $('html,body').animate({
                        scrollTop: target.offset().top
                    }, 1000);
                    return false;
                }
            }
        });
    });
    //#to-top button appears after scrolling
    var fixed = false;
    $(document).scroll(function() {
        if ($(this).scrollTop() > 250) {
            if (!fixed) {
                fixed = true;
                // $('#to-top').css({position:'fixed', display:'block'});
                $('#to-top').show("slow", function() {
                    $('#to-top').css({
                        position: 'fixed',
                        display: 'block'
                    });
                });
            }
        } else {
            if (fixed) {
                fixed = false;
                $('#to-top').hide("slow", function() {
                    $('#to-top').css({
                        display: 'none'
                    });
                });
            }
        }
    });
    // Disable Google Maps scrolling
    // See http://stackoverflow.com/a/25904582/1607849
    // Disable scroll zooming and bind back the click event
    var onMapMouseleaveHandler = function(event) {
        var that = $(this);
        that.on('click', onMapClickHandler);
        that.off('mouseleave', onMapMouseleaveHandler);
        that.find('iframe').css("pointer-events", "none");
    }
    var onMapClickHandler = function(event) {
            var that = $(this);
            // Disable the click handler until the user leaves the map area
            that.off('click', onMapClickHandler);
            // Enable scrolling zoom
            that.find('iframe').css("pointer-events", "auto");
            // Handle the mouse leave event
            that.on('mouseleave', onMapMouseleaveHandler);
        }
        // Enable map zooming with mouse scroll when the user clicks the map
    $('.map').on('click', onMapClickHandler);
    </script>
</body>
