<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>RECOMB 2017 - Satellite Programs</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/stylish-portfolio.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
</head>
<body>
	<a id="schedule-shortcut" href="seq-schedule.html" class="btn btn-dark btn-lg toggle"><i class="fa fa-calendar"></i></a>
	<!-- Navigation -->
    <a id="menu-toggle" href="#" class="btn btn-dark btn-lg toggle"><i class="fa fa-bars"></i></a>
    <nav id="sidebar-wrapper">
        <ul class="sidebar-nav">
            <a id="menu-close" href="#" class="btn btn-light btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
            <li>
                <a href="index.html" onclick=$("#menu-close").click();>RECOMB Home</a>
            </li>
            <li>
                <a href="index.html#about" onclick=$("#menu-close").click();>About</a>
            </li>
            <li>
                <a href="index.html#services" onclick=$("#menu-close").click();>Overview</a>
            </li>
            <li class="dropdown">
                <a href="registration.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Register <span class="caret"></span></a>
                <ul class="dropdown-menu" role="menu">
                        <li><a href="travelfellowship.html">Travel Fellowship</a></li>
                        <li><a href="accommodation.html">Hotel Information</a></li>
                </ul>
            </li>
                        <li>
                <a href="index.html#portfolio" onclick=$("#menu-close").click();>Submit to RECOMB</a>
            </li>
            <li class="dropdown">
                <a href="program.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Program <span class="caret"></span></a>
                <ul class="dropdown-menu" role="menu">
                        <li><a href="schedule.html">Schedule</a></li>
                        <li><a href="accepted_papers.html">Accepted Papers</a></li>
                        <li><a href="accepted_posters.html">Accepted Posters</a></li>
			<li><a href="awards.html">Award Winners</a></li>
                        <li><a href="useful_links.html">Useful Links</a></li>
                </ul>
            </li>
			<li>
                <a href="keynote.html" onclick=$("#menu-close").click();>Keynotes</a>
            </li>
			<li>
                <a href="committees.html" onclick=$("#menu-close").click();>Committees</a>
            </li>
		<li class="dropdown">
                <a href="satellites.html" onclick=$("#menu-close").click(); class="dropdown-hover" data-hover="dropdown">Satellites <span class="caret"></span></a>
                        <ul class="dropdown-menu" role="menu">
                        <li><a href="#">RECOMB-Seq</a></li>
                        <li><a href="http://computationalgenomics.bioinformatics.ucla.edu/2017-recomb-satellite-meetings/" target=_blank>RECOMB-Genetics</a></li>
                        <li><a href="http://computationalgenomics.bioinformatics.ucla.edu/2017-recomb-satellite-meetings/" target=_blank>RECOMB-CCB</a></li>
                </ul>
            </li>
            <li>
                <a href="index.html#contact" onclick=$("#menu-close").click();>Venue</a>
            </li>
		<li><br /><center><img src="img/logo-banner.png" alt="" width="95%"></center></li>
        </ul>
    </nav>
	<header class="main-header-seq">
		<div class="text-vertical-center">
            <font color=white>
			<h1>RECOMB-Seq 2017</h1>
            <h2>The Seventh RECOMB Satellite Workshop on Massively Parallel Sequencing</h2>
			<h3>Hong Kong, May 7-8, 2017</</h3>
			<!--<h2><font color="cyan">* Submissions are Open *</font></h2>-->
			</font>
        </div>
	</header>
	<!--Body-->
	<div class="container">
		<div class="row">
        	    <div class="col-lg-12">
                	<h1 class="page-header">RECOMB-Seq</br>
                	<small>The Seventh RECOMB Satellite Workshop on Massively Parallel Sequencing</small>
                	</h1>
            	    </div>
        	</div>
		<div class="row">
			<div class="col-md-8">
				<div class="row">
					<div class="col-md-12">
					<h3>Scope</h3>
					<p align="justify">The recent progress in sequencing technologies has opened the door to answering biological questions unimaginable a decade ago and as a result has revolutionized computational biology. The true potential of these technologies can be materialized only through advanced computational method development for analyzing these diverse high-throughput sequencing data. This workshop brings together leading researchers in computational genomics and genomic biology to discuss emerging frontiers in algorithm development for massively parallel sequencing data. The emphasis of all contributed work will be on applying statistical and algorithmic approaches to improve sequence data management, analysis, and interpretation.</p>
					</div>
				</div>
                                <div class="row">
                                        <div class="col-md-12">
					<h3>Contributions</h3>
					<p align="justify">We solicit contributions in four different categories as follows:
                                	<ol>
                                        	<li><p align="justify">Manuscripts describing original work on computational aspects of genomic research involving massively parallel sequencing.</p></li>
                                        	<li><p align="justify">Highlights Track abstracts. 1-2 pages describing computational aspects of work that has been published in a journal after <font color=red>May 1, 2016</font>, or papers that are <i>in press</i> at the time of submission and are already linked on the journal website. This track is joint with the main RECOMB conference.</p>
                                        	<li><p align="justify">Abstracts for short talks. 1-2 pages describing original work, including software applications. These will be considered for short presentations as well as for posters.</p></li>
                                      		<li><p align="justify">Posters.</p></li>
                                	</ol>
                                	</p>	
					</div>
                                </div>
				<div class="row">
					<div class="col-md-12">
	                                	<h3>Topics</h3>
        	                        	<p>(<i>include but not limited to</i>)</p>
                	                	<div class="row">
                        		                <div class="col-md-6">
                                			<ul>
                                        		<li>Read mapping for second and third generation sequencing technologies</li>
                                        		<li>Techniques for managing (e.g., data compression) and visualizing massive amount of sequencing data</li>
                                        		<li>Discovery and genotyping of genomic variants, including SNPs, indels, and structural variants</li>
                                        		<li>Local and de novo sequence assembly</li>
                                        		<li>RNA sequencing (RNAseq) methods, including the analysis of RNA expression and novel transcript assembly</li>
                                			</ul>
                        				</div>
                        				<div class="col-md-6">
                                			<ul>
                                        		<li>Methods for emerging sequencing technologies, such as single-cell or single molecule real time sequencing</li>
                                        		<li>Translational applications of sequencing data, including cancer genomics and infectious disease diagnostics and surveillance</li>
                                        		<li>Epigenetics and gene regulation, including ChIP-seq analysis, methylation profiling, and histone modification</li>
                                        		<li>Metagenomics</li>
	                                		</ul>
  	                      				</div>
        	                       		</div>
					</div>
                        	</div>
				<div class="row">
                	                <div class="col-md-12">					
						<h3>Paper Submission Procedures</h3>
						<p align="justify">
							Manuscripts should be no more than 10 single-spaced US letter or A4 pages with at most a 6.5x9" text area in at least 11 point font. Title and authors, corresponding author's email address, 100-250 word abstract, references, figures, and tables all included. An optional short appendix may contain details or additional data to be consulted at the discretion of the program committee. Manuscripts must be submitted electronically in PDF format via the <a href="https://easychair.org/conferences/?conf=recomb2017" target="_blank">EasyChair system</a>.
						</p>
						<p align="justify">
							Manuscripts submitted for review should represent original, previously unpublished work. At the time of submission to the workshop, and for the entire review period, the work should not be under review by any other conference or scientific journal.
						</p>
						<p align="justify">
							The submission system will be open starting <b><font color="red">January 3rd, 2017</font></b> and ending <b><font color="red"><strike>February 3rd, 2017</strike> February 5th, 2017</font></b>. Authors will be notified of acceptance or rejection by email on or before <b><font color="red">February 26th, 2017</font></b>. An author of each accepted paper is expected to attend the workshop and present the paper; otherwise, alternative arrangements should be made to have the paper presented.
						</p>
					</div>
				</div>
				<div class="row">
                	                <div class="col-md-12">
					<h3><i>Bioinformatics</i> Journal Partners with RECOMB-Seq Satellite Workshop</h3>
					<p align="justify">
					The Seventh RECOMB Satellite Workshop on Massively Parallel Sequencing (RECOMB-Seq) is continuing its successful partnership with <i>Bioinformatics</i>.  Selected papers accepted for the workshop will be considered for a special track publication in the journal <i>Bioinformatics</i>.
					</p>
					<p align="justify">
					<i>Bioinformatics</i> is a biweekly peer-reviewed scientific journal covering research and software in computational biology. The journal was established as Computer Applications in the Biosciences (CABIOS) in 1985. In 1998, the journal obtained its current name. <i>Bioinformatics</i> is published by Oxford University Press, and as of 2016, the editors-in-chief are Alfonso Valencia and Janet Kelso.
					</p>
					</div>
                                </div>
				<div class="row">
                                        <div class="col-md-12">
						<h3>Special Handling of Papers Submitted to RECOMB 2017</h3>
						<p align="justify">
                                		If you would like to consider submitting a paper rejected by RECOMB 2017 to the satellite workshop, you may do so by the regular deadline of <b><font color="red"><strike>February 3rd</strike> February 5th</font></b>. You must include in your submission a <i>Cover Letter</i> that includes the reviews received from RECOMB together with a rebuttal addressing the concerns raised by the reviewers. The submitted manuscript should also be modified according to the reviews, or provide an explanation of why modifications are unnecessary. All reviews will be treated confidentially and will only be visible to the PC members evaluating your paper.
						</p>
					</div>
                                </div>
				<div class="row">
					<div id="accordion" role="tablist" aria-multiselectable="true" class="col-lg-12">
                        			<div class="card">
                                			<div class="card-header" role="tab" id="headingOne">
                                				<h3 class="mb-0"><a class="btn btn-dark btn-lg" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
                                 				Accepted Regular Talks</a></h3>
                                			</div>
                                			<div id="collapseOne" class="collapse" role="tabpanel" aria-labelledby="headingOne">
                                			<div class="card-block">
							<ol>
							<li><p align="justify">Pavel Skums, Alex Zelikovsky, Sergey Knyazev, Igor Mandric, Zoya Dimitrova, Sumathi Ramachandran, David Stiven Campo Rendon, Leonid Bunimovich, Elizabeth Costenbader, Connie Sexton, Siobhan O'Connor, Guo-Liang Xia and Yury Khudyakov. <b> QUENTIN: accurate reconstruction of disease transmissions from viral quasispecies genomic data</b></p></li>
							<li><p align="justify">Vikas Bansal. <b> An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments</b></p></li>
							<li><p align="justify">Xinan Liu, Ye Yu, Jinpeng Liu, Chen Qian and Jinze Liu. <b> A Novel Data Structure to Support Ultra-fast Taxonomic Classification of Metagenomic Sequences with k-mer Signatures</b></p></li>
							<li><p align="justify">Zhe Sun, Ting Wang, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu and Wei Chen. <b> DIMM-SC: A Dirichlet mixture model for clustering droplet-based single cell transcriptomic data</b></p></li>
							<li><p align="justify">Luca Mariani, Kathryn Weinand, Anastasia Vedenko, Luis A. Barrera and Martha L. Bulyk. <b>Transcription factor-8mer glossary with GENRE genomic background enables precise identification of lineage-specific co-regulators</b></p></li>
							<li><p align="justify">Dilip Durai and Marcel Schulz. <b> In-silico read normalization using set multicover optimization</b></p></li>
							<li><p align="justify">Igor Mandric, Sergey Knyazev and Alex Zelikovsky. <b> Repeat-aware evaluation of scaffolding tools</b></p></li>
							<li><p align="justify">Roye Rozov, Gil Goldshlager, Ron Shamir and Eran Halperin. <b> Faucet: streaming de novo assembly graph construction</b></p></li>
							</ol>
							</div>
							</div>
							<div class="card-header" role="tab" id="headingTwo">
                                        			<h3 class="mb-0"><a class="btn btn-dark btn-lg" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                                                		Accepted Short Talks</a></h3>
                                        		</div>
							<div id="collapseTwo" class="collapse" role="tabpanel" aria-labelledby="headingTwo">
                                        		<div class="card-block">
                                                	<ol>
							<li><p align="justify">Tony Ginart, Kaiyuan Zhu, Joseph Hui, Ibrahim Numanagic, David Tse, Thomas Courtade and Cenk Sahinalp. <b> Genomic Reads Forests for Compressed Representation of High Throughput Sequence Data</b></p></li>
							<li><p align="justify">Massimo Cairo, Paul Medvedev, Nidia Obscura Acosta, Romeo Rizzi and Alexandru I. Tomescu. <b>Faster Omnitig Listing for Safe and Complete Contig Assembly</b></p></li>
							<li><p align="justify">Rahul Nihalani, Sriram P. Chockalingam, Shaowei Zhu, Vijay Vazirani and Srinivas Aluru. <b>Probabilistic Estimation of Overlap Graphs for Large Sequence Datasets</b></p></li>
							<li><p align="justify">Xintong Chen, Oscar Rodriguez, Matthew Pendleton, Bojan Losic and Ali Bashir. <b> TransPac: transposon detection and characterization from long-reads</b></p></li>
							<li><p align="justify">Jens Quedenfeld and Sven Rahmann. <b> Variant tolerant read mapping using min-hashing</b></p></li>
							<li><p align="justify">Evgenii I. Olekhnovich, Artem T. Vasilyev, Vladimir I. Ulyantsev and Alexander V. Tyakht. <b>MetaCherchant – an algorithm for analyzing genomic environment of antibiotic resistance gene in gut microbiota</b></p></li>
							<li><p align="justify">Sarah Yeo, Lauren Coombe, Justin Chu, Rene Warren and Inanc Birol. <b> ARCS: Assembly Roundup by Chromium Scaffolding</b></p></li>
							<li><p align="justify">Martin Muggli, Simon Puglisi and Christina Boucher. <b> Kohdista: A Succinct Solution to Rmap Alignment</b></p></li>
							<li><p align="justify">Abolfazl Hashemi, Banghua Zhu and Haris Vikalo. <b> A Tensor Factorization Framework for Haplotype Assembly of Diploids and Polyploids</b></p></li>
							<li><p align="justify">Hongyi Xin, Jeremie Kim, Sunny Nahar, Can Alkan and Onur Mutlu. <b> LEAP: A Generalization of the Landau-Vishkin Algorithm with Custom Gap Penalties</b></p></li>
							<li><p align="justify">Mingfu Shao and Carl Kingsford. <b> Theory and Algorithm for the Minimum Path Flow Decomposition Problem</b></p></li>
                                                	</ol>
                                        		</div>
                                       			 </div>
						</div>
                                        </div>
                                </div>
			</div>
			<div class="col-md-4">
			<div class="row">
			<div class="col-md-12">
					<h3>Schedule At-a-Glance</h3>
                        		<p><i>Full schedule <a href="seq-schedule.html">here</a>.</i></p>
                        		<table>
                        		<tr>
                        		<td><b>Sunday, May 7</b></td>
                        		</tr>
                        		<tr>
                        		<td>1:00 PM - 2:00 PM
                        		</td>
                        		<td>
                        		&nbsp;&nbsp;Resgistration
                        		</td>
                        		</tr>
                        		<tr>
                        		<td>2:00 PM - 3:00 PM
                        		</td>
                        		<td>
                        		&nbsp;&nbsp;KT - Olga Troyanskaya
                        		</td>
                        		</tr>
                        		<tr>
                        		<td>3:00 PM - 3:20 PM
                        		</td>
                        		<td>
                        		&nbsp;&nbsp;Coffee Break
                        		</td>
                        		</tr>
                        		<tr>
                        		<td>3:20 PM - 4:20 PM
                        		</td>
                        		<td>
                       			 &nbsp;&nbsp;RT - Skums et al., Liu et al.,
                        		</td>
                        		</tr>
					<tr>
	                        <td></td>
	                        <td>&nbsp;&nbsp;Mariani et al.</td>
	                        </tr>
	                        <tr>
	                        <td>4:20 PM - 4:30 PM
	                        </td>
	                        <td>
	                        &nbsp;&nbsp;Break
	                        </td>
	                        </tr>
	                        <tr>
                        <td>4:30 PM - 5:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;ST - Ginart et al.,
                        </td>
                        </tr>
                        <tr>
                        <td></td>
                        <td>&nbsp;&nbsp;Xin et al., Shao et al.</td>
                        </tr>
                        <tr>
                        <td><b>Monday, May 8</b>
                        </td>
                        </tr>
                        <tr>
                        <td>8:00 AM - 9:00 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;Registration
                        </td>
                        </tr>
                        <tr>
                        <td>9:00 AM - 10:00 AM
			</td>
                        <td>
                        &nbsp;&nbsp;KT - Sharon Aviran
                        </td>
                        </tr>
                        <tr>
                        <td>10:00 AM - 10:20 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;Coffee Break
                        </td>
                        </tr>
                        <tr>
                        <td>10:20 AM - 11:20 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;RT - Rozov et al., Sun et al.
                        </td>
                        </tr>
                        <tr>
                        <td></td>
                        <td>
                        &nbsp;&nbsp;Durai et al.
                        </td>
                        </tr>
                        <tr>
                        <td>11:20 AM - 11:30 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;Break
                        </td>
                        </tr>
                        <tr>
                        <td>11:30 AM - 12:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;ST - Cairo et al., Nihalani et al.,
                        </td>
                        </tr>
                        <tr>
                        <td></td>
                        <td>
                        &nbsp;&nbsp;Chen et al., Quendenfeld et al.,
                        </td>
                        </tr>
                        <tr>
                        <td></td>
                        <td>
                        &nbsp;&nbsp;Yeo et al., Hashemi et al.
                        </td>
                        </tr>
                        <tr>
                        <td>12:30 PM - 2:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Lunch Break
                        </td>
                        </tr>
                        <tr>
                        <td>2:00 PM - 3:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;RT - Mandric et al., Bansal
                        </td>
                        </tr>
                        <tr>
                        <td>3:00 PM - 3:20 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Coffee Break
                        </td>
                        </tr>
                        <tr>
                        <td>3:20 PM - 4:20 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Poster Session I
                        </td>
                        </tr>
                        <tr>
                        <td>4:20 PM - 4:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Break
                        </td>
                        </tr>
                        <tr>
                        <td>4:30 PM - 5:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Poster Session II
                        </td>
                        </tr>
                        		</table>
			</div>
			</div>	
			<div class="row">
			<div class="col-md-12">
                                <h3>Key Dates</h3>
                                <p>
                                <b>February 3:</b> Paper submission deadline <b><font color=red>extended till February 5</font></b></br>
                                <b>February 5:</b> Reviewers assigned</br>
                                <b>February 26:</b> Reviews due & poster submission deadline</br>
                                <b>March 10:</b> Program announced</br>
                                <b>May 7-8:</b> Conference</br>
                                </p>
                        </div>
			</div>
			<div class="row">
                        <div class="col-md-12">
				<h3>Invited Speakers</h3>
				<p>
                                <b>Olga Troyanskaya</b><br />
                                Department of Computer Science<br/>
                                Lewis-Sigler Institute of Integrative Genomics<br/>
                                Princeton University
                        	</p>
                        	<p>
                                <b>Sharon Aviran</b><br/>
                                Department of Biomedical Engineering<br/>
                                The Genome Center<br/>
                                University of California, Davis
                        	</p>
                        </div>
                        </div>
			<div class="row">
                        <div class="col-md-12">
                                <h3>Organizing Committee</h3>
	                        <p>
	                        <b>Program Committee Chairs</b>
        	                <ul>
                                <li>Fereydoun Hormozdiari, UC Davis, USA</li>
                                <li>Jian Ma, Carnegie Mellon University, USA</li>
                        </ul>
                        </p>
                        <p>
                        <b>Steering Committee</b>
                        <ul>
                                <li>Inanc Birol, Michael Smith Genome Sciences Centre, Canada</li>
                                <li>Michael Brudno, University of Toronto, Canada</li>
                                <li>Eran Halperin, UCLA, USA</li>
                                <li>Ben Raphael, Princeton University, USA</li>
                                <li>Cenk Sahinalp, Indiana University, Bloomington, USA</li>
                        </ul>
                        </p>
                        <p>
                        <b>Program Committee</b>
                        <ul>
                                <li>Can Alkan, Bilkent University</li>
                                <li>Lars Arvestad, Stockholm University</li>
                                <li>Sharon Aviran, UC Davis</li>
                                <li>Niko Beerenwinkel, ETH Zurich</li>
                                <li>Titus Brown, UC Davis</li>
                                <li>Mark Chaisson, University of Washington</li>
                                <li>Ken Chen, MD Anderson</li>
                                <li>Melissa Gymrek, UCSD</li>
                                <li>Faraz Hach, Simon Fraser University</li>
                                <li>Iman Hajirasouliha, Weill Cornell Medical College</li>
                                <li>Michael Hoffman, Princess Margaret Cancer Centre</li>
                                <li>Farhad Hormozdiari, Harvard Medical School</li>
                                <li>Peter Kharchenko, Harvard Medical School</li>
                                <li>Carl Kingsford, Carnegie Mellon University</li>
                                <li>Gunnar Klau, VU Amsterdam</li>
                                <li>Ben Langmead, Johns Hopkins University</li>
                                <li>Wenxiu Ma, UC Riverside</li>
                                <li>Veli M&auml;kinen, University of Helsinki</li>
                                <li>Serghei Mangul, UCLA</li>
                                <li>Tobias Marschall, Max Planck Institute for Informatics</li>
                                <li>Paul Medvedev, Penn State University</li>
                                <li>Olgica Milenkovic, University of Illinois at Urbana-Champaign</li>
                                <li>Niranjan Nagarajan, Genome Institute of Singapore</li>
				<li>Robert Patro, SUNY Stony Brook</li>
                                <li>Teresa Przytycka, National Institutes of Health</li>
                                <li>Sven Rahmann, University of Duisburg-Essen</li>
                                <li>Michael Schatz, Johns Hopkins University</li>
                                <li>Alexander Schliep, Rutgers University</li>
                                <li>Russell Schwartz, Carnegie Mellon University</li>
                                <li>Jared Simpson, OICR</li>
                                <li>Wing-Kin Sung, National University of Singapore</li>
                                <li>James Taylor, Johns Hopkins University</li>
                                <li>Fabio Vandin, University of Padova</li>
                                <li>Jianyang Zeng, Tsinghua University</li>
                                <li>Fangqing Zhao, Chinese Academy of Sciences</li>
                        </ul>
                        </p>
			</div>
                        </div>
		</div>
	</div>
<!--				
			<div class="col-lg-11">
				<h3>Topics</h3>
				<p>(<i>include but not limited to</i>)</p>	
			</div>
			<div class="col-md-6">
				<ul>
					<li>Read mapping for second and third generation sequencing technologies</li>
					<li>Techniques for managing (e.g., data compression) and visualizing massive amount of sequencing data</li>
					<li>Discovery and genotyping of genomic variants, including SNPs, indels, and structural variants</li>
					<li>Local and de novo sequence assembly</li>
					<li>RNA sequencing (RNAseq) methods, including the analysis of RNA expression and novel transcript assembly</li>
				</ul>
			</div>
			<div class="col-md-5">
				<ul>
					<li>Methods for emerging sequencing technologies, such as single-cell or single molecule real time sequencing</li>
					<li>Translational applications of sequencing data, including cancer genomics and infectious disease diagnostics and surveillance</li>
					<li>Epigenetics and gene regulation, including ChIP-seq analysis, methylation profiling, and histone modification</li>
					<li>Metagenomics</li>
				</ul>
			</div>
			
			<div class="col-lg-11">
			<h3>Paper Submission Procedures</h3>
			</div>
			<div class="col-md-11">
				<p align="justify">
				Manuscripts should be no more than 10 single-spaced US letter or A4 pages with at most a 6.5x9" text area in at least 11 point font. Title and authors, corresponding author's email address, 100-250 word abstract, references, figures, and tables all included. An optional short appendix may contain details or additional data to be consulted at the discretion of the program committee. Manuscripts must be submitted electronically in PDF format via the <a href="https://easychair.org/conferences/?conf=recomb2017" target=_blank>EasyChair system</a>.
				</p>
				<p align="justify">
				Manuscripts submitted for review should represent original, previously unpublished work. At the time of submission to the workshop, and for the entire review period, the work should not be under review by any other conference or scientific journal.
				</p>
				<p align="justify">
				The submission system will be open starting <b><font color=red>January 3rd, 2017</font></b> and ending <b><font color=red><strike>February 3rd, 2017</strike> February 5th, 2017</font></b>. Authors will be notified of acceptance or rejection by email on or before <b><font color=red>February 26th, 2017</font></b>. An author of each accepted paper is expected to attend the workshop and present the paper; otherwise, alternative arrangements should be made to have the paper presented.
				</p>
				
			</div>
			<div class="col-lg-11">
			<h3><i>Bioinformatics</i> Journal Partners with RECOMB-Seq Satellite Workshop</h3>
			</div>
			<div class="col-md-11">
				<p align="justify">
				The Seventh RECOMB Satellite Workshop on Massively Parallel Sequencing (RECOMB-Seq) is continuing its successful partnership with <i>Bioinformatics</i>.  Selected papers accepted for the workshop will be considered for a special track publication in the journal <i>Bioinformatics</i>.
				</p>
				<p align="justify">
				<i>Bioinformatics</i> is a biweekly peer-reviewed scientific journal covering research and software in computational biology. The journal was established as Computer Applications in the Biosciences (CABIOS) in 1985. In 1998, the journal obtained its current name. <i>Bioinformatics</i> is published by Oxford University Press, and as of 2016, the editors-in-chief are Alfonso Valencia and Janet Kelso.
				</p>
			</div>
			<div class="col-lg-11">
                        <h3>Special Handling of Papers Submitted to RECOMB 2017</h3>
                        </div>
                        <div class="col-md-11">
                                <p align="justify">
                                If you would like to consider submitting a paper rejected by RECOMB 2017 to the satellite workshop, you may do so by the regular deadline of <b><font color="red"><strike>February 3rd</strike> February 5th</font></b>. You must include in your submission a <i>Cover Letter</i> that includes the reviews received from RECOMB together with a rebuttal addressing the concerns raised by the reviewers. The submitted manuscript should also be modified according to the reviews, or provide an explanation of why modifications are unnecessary. All reviews will be treated confidentially and will only be visible to the PC members evaluating your paper.
				</p>
                        </div>
			<div id="accordion" role="tablist" aria-multiselectable="true" class="col-lg-11">
                        	<div class="card">
                                	<div class="card-header" role="tab" id="headingOne">
                                	<h3 class="mb-0"><a class="btn btn-dark btn-lg" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
                                 		Accepted Regular Talks</a></h3>
                                	</div>
                                	<div id="collapseOne" class="collapse" role="tabpanel" aria-labelledby="headingOne">
                                	<div class="card-block">
						<ol>
							<li><p align="justify">Pavel Skums, Alex Zelikovsky, Sergey Knyazev, Igor Mandric, Zoya Dimitrova, Sumathi Ramachandran, David Stiven Campo Rendon, Leonid Bunimovich, Elizabeth Costenbader, Connie Sexton, Siobhan O'Connor, Guo-Liang Xia and Yury Khudyakov. <b> QUENTIN: accurate reconstruction of disease transmissions from viral quasispecies genomic data</b></p></li>
							<li><p align="justify">Vikas Bansal. <b> An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments</b></p></li>
							<li><p align="justify">Xinan Liu, Ye Yu, Jinpeng Liu, Chen Qian and Jinze Liu. <b> A Novel Data Structure to Support Ultra-fast Taxonomic Classification of Metagenomic Sequences with k-mer Signatures</b></p></li>
							<li><p align="justify">Zhe Sun, Ting Wang, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu and Wei Chen. <b> DIMM-SC: A Dirichlet mixture model for clustering droplet-based single cell transcriptomic data</b></p></li>
							<li><p align="justify">Luca Mariani, Kathryn Weinand, Anastasia Vedenko, Luis A. Barrera and Martha L. Bulyk. <b>Transcription factor-8mer glossary with GENRE genomic background enables precise identification of lineage-specific co-regulators</b></p></li>
							<li><p align="justify">Dilip Durai and Marcel Schulz. <b> In-silico read normalization using set multicover optimization</b></p></li>
							<li><p align="justify">Igor Mandric, Sergey Knyazev and Alex Zelikovsky. <b> Repeat-aware evaluation of scaffolding tools</b></p></li>
							<li><p align="justify">Roye Rozov, Gil Goldshlager, Ron Shamir and Eran Halperin. <b> Faucet: streaming de novo assembly graph construction</b></p></li>
						</ol>
					</div>
					</div>
					<div class="card-header" role="tab" id="headingTwo">
                                        <h3 class="mb-0"><a class="btn btn-dark btn-lg" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                                                Accepted Short Talks</a></h3>
                                        </div>
					<div id="collapseTwo" class="collapse" role="tabpanel" aria-labelledby="headingTwo">
                                        <div class="card-block">
                                                <ol>
							<li><p align="justify">Tony Ginart, Kaiyuan Zhu, Joseph Hui, Ibrahim Numanagic, David Tse, Thomas Courtade and Cenk Sahinalp. <b> Genomic Reads Forests for Compressed Representation of High Throughput Sequence Data</b></p></li>
							<li><p align="justify">Massimo Cairo, Paul Medvedev, Nidia Obscura Acosta, Romeo Rizzi and Alexandru I. Tomescu. <b>Faster Omnitig Listing for Safe and Complete Contig Assembly</b></p></li>
							<li><p align="justify">Rahul Nihalani, Sriram P. Chockalingam, Shaowei Zhu, Vijay Vazirani and Srinivas Aluru. <b>Probabilistic Estimation of Overlap Graphs for Large Sequence Datasets</b></p></li>
							<li><p align="justify">Xintong Chen, Oscar Rodriguez, Matthew Pendleton, Bojan Losic and Ali Bashir. <b> TransPac: transposon detection and characterization from long-reads</b></p></li>
							<li><p align="justify">Jens Quedenfeld and Sven Rahmann. <b> Variant tolerant read mapping using min-hashing</b></p></li>
							<li><p align="justify">Evgenii I. Olekhnovich, Artem T. Vasilyev, Vladimir I. Ulyantsev and Alexander V. Tyakht. <b>MetaCherchant – an algorithm for analyzing genomic environment of antibiotic resistance gene in gut microbiota</b></p></li>
							<li><p align="justify">Sarah Yeo, Lauren Coombe, Justin Chu, Rene Warren and Inanc Birol. <b> ARCS: Assembly Roundup by Chromium Scaffolding</b></p></li>
							<li><p align="justify">Martin Muggli, Simon Puglisi and Christina Boucher. <b> Kohdista: A Succinct Solution to Rmap Alignment</b></p></li>
							<li><p align="justify">Abolfazl Hashemi, Banghua Zhu and Haris Vikalo. <b> A Tensor Factorization Framework for Haplotype Assembly of Diploids and Polyploids</b></p></li>
							<li><p align="justify">Hongyi Xin, Jeremie Kim, Sunny Nahar, Can Alkan and Onur Mutlu. <b> LEAP: A Generalization of the Landau-Vishkin Algorithm with Custom Gap Penalties</b></p></li>
							<li><p align="justify">Mingfu Shao and Carl Kingsford. <b> Theory and Algorithm for the Minimum Path Flow Decomposition Problem</b></p></li>
                                                </ol>
                                        </div>
                                        </div>
				</div>
			</div>
		</div>
		<div style="float:right; width:30%">
			<div class="col-lg-12">
			<h3>Schedule At-a-Glance</h3>
			<p><i>Full schedule <a href="seq-schedule.html">here</a>.</i></p>
			<table>
			<tr>
			<td><b>Sunday, May 7</b></td>
			</tr>
			<tr>
			<td>1:00 PM - 2:00 PM
			</td>
			<td>
			&nbsp;&nbsp;Resgistration
			</td>
			</tr>
			<tr>
			<td>2:00 PM - 3:00 PM
			</td>
			<td>
			&nbsp;&nbsp;KT - Olga Troyanskaya
			</td>
			</tr>
			<tr>
			<td>3:00 PM - 3:20 PM
			</td>
			<td>
			&nbsp;&nbsp;Coffee Break
			</td>
			</tr>
			<tr>
                        <td>3:20 PM - 4:20 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;RT - Skums et al., Liu et al.,
                        </td>
                        </tr>
			<tr>
			<td></td>
			<td>&nbsp;&nbsp;Mariani et al.</td>
			</tr>
			<tr>
                        <td>4:20 PM - 4:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Break
                        </td>
                        </tr>
			<tr>
                        <td>4:30 PM - 5:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;ST - Ginart et al.,
                        </td>
                        </tr>
			<tr>
                        <td></td>
                        <td>&nbsp;&nbsp;Xin et al., Shao et al.</td>
                        </tr>
			<tr>
                        <td><b>Monday, May 8</b>
                        </td>
			</tr>
			<tr>
			<td>8:00 AM - 9:00 AM
			</td>
                        <td>
                        &nbsp;&nbsp;Registration
                        </td>
                        </tr>
			<tr>
                        <td>9:00 AM - 10:00 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;KT - Sharon Aviran
                        </td>
                        </tr>
			<tr>
                        <td>10:00 AM - 10:20 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;Coffee Break
                        </td>
                        </tr>
			<tr>
                        <td>10:20 AM - 11:20 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;RT - Rozov et al., Sun et al.
                        </td>
                        </tr>
			<tr>
			<td></td>
			<td>
                        &nbsp;&nbsp;Durai et al.
                        </td>
			</tr>
			<tr>
                        <td>11:20 AM - 11:30 AM
                        </td>
                        <td>
                        &nbsp;&nbsp;Break       
                        </td>
                        </tr>
			<tr>
                        <td>11:30 AM - 12:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;ST - Cairo et al., Nihalani et al.,
                        </td>
                        </tr>
                        <tr>
                        <td></td>
                        <td>
                        &nbsp;&nbsp;Chen et al., Quendenfeld et al.,
                        </td>
                        </tr>
			<tr>
                        <td></td>
                        <td>
                        &nbsp;&nbsp;Yeo et al., Hashemi et al.
                        </td>
                        </tr>
			<tr>
                        <td>12:30 PM - 2:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Lunch Break       
                        </td>
                        </tr>
			<tr>
                        <td>2:00 PM - 3:00 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;RT - Mandric et al., Bansal
                        </td>
                        </tr>
			<tr>
                        <td>3:00 PM - 3:20 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Coffee Break
                        </td>
                        </tr>
			<tr>
                        <td>3:20 PM - 4:20 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Poster Session I
                        </td>
                        </tr>
			<tr>
                        <td>4:20 PM - 4:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Break
                        </td>
                        </tr>
			<tr>
                        <td>4:30 PM - 5:30 PM
                        </td>
                        <td>
                        &nbsp;&nbsp;Poster Session II
                        </td>
                        </tr>
			</table>
			</div>
			<div class="col-lg-12">
				<h3>Key Dates</h3>
			</div>
			<div class="col-md-12">
			<p>
				<b>February 3:</b> Paper submission deadline <b><font color=red>extended till February 5</font></b></br>
				<b>February 5:</b> Reviewers assigned</br>
				<b>February 26:</b> Reviews due & poster submission deadline</br>
				<b>March 10:</b> Program announced</br>
				<b>May 7-8:</b> Conference</br>
			</p>
			</div>
			<div class="col-lg-12">
				<h3>Invited Speakers</h3>
			</div>
			<div class="col-md-12">
			<p>
				<b>Olga Troyanskaya</b><br />
				Department of Computer Science<br/>
				Lewis-Sigler Institute of Integrative Genomics<br/>
				Princeton University
			</p>
			<p>
                                <b>Sharon Aviran</b><br/>
                                Department of Biomedical Engineering<br/>
                                The Genome Center<br/>
                                University of California, Davis
                        </p>
			</div>
			<div class="col-lg-12">
				<h3>Organizing Committee</h3>
			</div>
			<div class="col-md-12">
			<p>
			<b>Program Committee Chairs</b>
			<ul>
				<li>Fereydoun Hormozdiari, UC Davis, USA</li>
				<li>Jian Ma, Carnegie Mellon University, USA</li>
			</ul>
			</p>
			<p>
			<b>Steering Committee</b>
			<ul>
				<li>Inanc Birol, Michael Smith Genome Sciences Centre, Canada</li>
				<li>Michael Brudno, University of Toronto, Canada</li>
				<li>Eran Halperin, UCLA, USA</li>
				<li>Ben Raphael, Princeton University, USA</li>
				<li>Cenk Sahinalp, Indiana University, Bloomington, USA</li>
			</ul>
			</p>
			<p>
			<b>Program Committee</b>
			<ul>
				<li>Can	Alkan, Bilkent University</li>
				<li>Lars Arvestad, Stockholm University</li>
				<li>Sharon Aviran, UC Davis</li>
				<li>Niko Beerenwinkel, ETH Zurich</li>
				<li>Titus Brown, UC Davis</li>
				<li>Mark Chaisson, University of Washington</li>
				<li>Ken Chen, MD Anderson</li>
				<li>Melissa Gymrek, UCSD</li>
				<li>Faraz Hach, Simon Fraser University</li>
				<li>Iman Hajirasouliha, Weill Cornell Medical College</li>
				<li>Michael Hoffman, Princess Margaret Cancer Centre</li>
				<li>Farhad Hormozdiari, Harvard Medical School</li>
				<li>Peter Kharchenko, Harvard Medical School</li>
				<li>Carl Kingsford, Carnegie Mellon University</li>
				<li>Gunnar Klau, VU Amsterdam</li>
				<li>Ben Langmead, Johns Hopkins University</li>
				<li>Wenxiu Ma, UC Riverside</li>
				<li>Veli M&auml;kinen, University of Helsinki</li>
				<li>Serghei Mangul, UCLA</li>
				<li>Tobias Marschall, Max Planck Institute for Informatics</li>
				<li>Paul Medvedev, Penn State University</li>
				<li>Olgica Milenkovic, University of Illinois at Urbana-Champaign</li>
				<li>Niranjan Nagarajan, Genome Institute of Singapore</li>
				<li>Robert Patro, SUNY Stony Brook</li>
				<li>Teresa Przytycka, National Institutes of Health</li>
				<li>Sven Rahmann, University of Duisburg-Essen</li>
				<li>Michael Schatz, Johns Hopkins University</li>
				<li>Alexander Schliep, Rutgers University</li>
				<li>Russell Schwartz, Carnegie Mellon University</li>
				<li>Jared Simpson, OICR</li>
				<li>Wing-Kin Sung, National University of Singapore</li>
				<li>James Taylor, Johns Hopkins University</li>
				<li>Fabio Vandin, University of Padova</li>
				<li>Jianyang Zeng, Tsinghua University</li>
				<li>Fangqing Zhao, Chinese Academy of Sciences</li>
			</ul>
			</p>
			</div>
		</div>
		</div>
	</div>-->
	<footer>
        <div class="container">
            <div class="row">
                <div class="col-lg-10 col-lg-offset-1 text-center">
                    <hr class="small">
                    <p class="text-muted">Copyright &copy; RECOMB 2017 | Theme: Bootstrap Stylish Portfolio | Design: Sumaiya Nazeen</p>
                </div>
            </div>
        </div>
        <a id="to-top" href="#top" class="btn btn-dark btn-lg"><i class="fa fa-chevron-up fa-fw fa-1x"></i></a>
    </footer>
	<!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Custom Theme JavaScript -->
    <script>
	$('ul.sidebar-nav li.dropdown').hover(function() {
	  $(this).find('.dropdown-menu').stop(true, true).delay(200).fadeIn(500);
	}, function() {
	  $(this).find('.dropdown-menu').stop(true, true).delay(200).fadeOut(500);
	});
    </script>
    <script>
    // Closes the sidebar menu
    $("#menu-close").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });
    // Opens the sidebar menu
    $("#menu-toggle").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });
    // Scrolls to the selected menu item on the page
    $(function() {
        $('a[href*=#]:not([href=#],[data-toggle],[data-target],[data-slide])').click(function() {
            if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
                var target = $(this.hash);
                target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
                if (target.length) {
                    $('html,body').animate({
                        scrollTop: target.offset().top
                    }, 1000);
                    return false;
                }
            }
        });
    });
    //#to-top button appears after scrolling
    var fixed = false;
    $(document).scroll(function() {
        if ($(this).scrollTop() > 250) {
            if (!fixed) {
                fixed = true;
                // $('#to-top').css({position:'fixed', display:'block'});
                $('#to-top').show("slow", function() {
                    $('#to-top').css({
                        position: 'fixed',
                        display: 'block'
                    });
                });
            }
        } else {
            if (fixed) {
                fixed = false;
                $('#to-top').hide("slow", function() {
                    $('#to-top').css({
                        display: 'none'
                    });
                });
            }
        }
    });
    // Disable Google Maps scrolling
    // See http://stackoverflow.com/a/25904582/1607849
    // Disable scroll zooming and bind back the click event
    var onMapMouseleaveHandler = function(event) {
        var that = $(this);
        that.on('click', onMapClickHandler);
        that.off('mouseleave', onMapMouseleaveHandler);
        that.find('iframe').css("pointer-events", "none");
    }
    var onMapClickHandler = function(event) {
            var that = $(this);
            // Disable the click handler until the user leaves the map area
            that.off('click', onMapClickHandler);
            // Enable scrolling zoom
            that.find('iframe').css("pointer-events", "auto");
            // Handle the mouse leave event
            that.on('mouseleave', onMapMouseleaveHandler);
        }
        // Enable map zooming with mouse scroll when the user clicks the map
    $('.map').on('click', onMapClickHandler);
    </script>
</body>
